### DISCUSSION TITLE
Docs: Clarification regarding userless sign in

### DISCUSSION THREAD

**Discussion by thannaske** [rawilk/laravel-webauthn#14] [Q&A]
In the advanced documentation there is a [section about userless sign in](https://randallwilk.dev/docs/laravel-webauthn/v1/advanced-usage/advanced-configuration#user-content-userless-login). However, a clarification in terms how to implement it exactly, would be a great addition to the docs.

I tried to implement it, however the `PrepareAssertionData::class` requires a challenged user as argument and when we're performing a userless sign in we don't have a challenged user, yet. Could you give me a pointer on how to accomplish the userless login. The signage on the client side requires a public key to be signed and without the challenged user I have no idea what to give the user to sign. Could this be an arbitrary public key just for the purpose to get the user to sign something to check which key he's using?

Any help is highly appreciated!

**Comment by rawilk** [comment]
Converted this to a discussion as I think it's better suited here.
